src/Mesh/Grids.jl

```diff
@@ -231,4 +231,134 @@ function computegeometry(topology::AbstractTopology{dim}, D, ξ, ω, meshwarp,
 end
 # }}}
 
+
+# {{{ compute element size for box domain 
+function compute_element_size(dim, Nq, vgeo, e) 
+
+    DFloat = eltype(vgeo)
+    
+    if (dim == 2)
+        
+        x, y = zeros(DFloat, 4), zeros(DFloat, 4)
+        
+        x[1], y[1] = vgeo[1, 1,   _x, e], vgeo[1, 1,   _y, e]
+        x[2], y[2] = vgeo[Nq, 1,  _x, e], vgeo[Nq, 1,  _y, e]
+        x[3], y[3] = vgeo[1, Nq,  _x, e], vgeo[1, Nq,  _y, e]
+        x[4], y[4] = vgeo[Nq, Nq, _x, e], vgeo[Nq, Nq, _y, e]        
+
+        #Element sizes (as if it were linear)
+        dx = maximum(x[:]) - minimum(x[:])
+        dy = maximum(y[:]) - minimum(y[:])
+
+        
+        #Average distance between LGL points inside the element:
+        dx_mean = dx/max(Nq - 1, 1)
```

Comment:
mean dof distance isn't really part of "element size", so not sure this should be here.

Reply:
I don't understand what you mean by this.

Reply:
When I think of element size I do not think of average degree of freedom spacing. The element is the same size for polynomial order 1 and polynomial order 10

Reply:
I see what you mean. I will think of a different structure. Let me know if you have an idea for this. I need to think of how or where to put it.

Reply:
I don't feel comfortable removing this part since I did not work on LowStorageRungeKuttaMethod.jl Please, advise.

Reply:
Why not put it where you need it?